app/src/main/java/waiter/Interactor/Interactive.java

```diff
@@ -0,0 +1,14 @@
+package waiter.Interactor;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public interface Interactive {
```

Comment:
This is the wrapper for the `Socket` Java Class

Interactive was the name chosen as this is what the user uses to interact with the Server